docs/docs/91-migrations.md

```diff
@@ -4,9 +4,10 @@ Some versions need some changes to the server configuration or the pipeline conf
 
 ## `next`
 
+- Rename server environment variable `WOODPECKER_ESCALATE` to `WOODPECKER_PLUGINS_PRIVILEGED`
+- Remove all default privileged plugins ([re-add plugins to the list via config if needed](./30-administration/10-server-config.md#woodpecker_plugins_privileged)).
```

Comment:
This is confusing as they should not be removed by the admin. Please adjust accordingly

Reply:
? what exactly do you mean?

Reply:
The migrations list is containing steps the admin is supposed to do.  ` Remove all plugins` doesn't make sense as we already removed all plugins. What you probably want to say is something like `All privileged plugins were removed. Add those like buildx which are required by your setup`. In general provide some more details for such a breaking change.

Reply:
Uh sure >D

Reply:
-> https://github.com/woodpecker-ci/woodpecker/pull/4088

Reply:
thanks